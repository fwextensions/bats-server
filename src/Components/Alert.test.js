import React from 'react';
import renderer from 'react-test-renderer';
import Alert from './Alert';

test('Snapshot', () => {
  const component = renderer.create(
    <Alert
      type="warning"
      title="Update divert status?"
      message="Ambulances will be notified."
      cancel="Keep status"
      destructive="Update status"
      onDestructive={() => {}}
      onCancel={() => {}}
    />);

    const componentJson = component.toJSON();
    expect(componentJson).toMatchSnapshot();
});