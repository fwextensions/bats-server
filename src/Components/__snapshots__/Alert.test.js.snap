// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div
  className="alert"
>
  <div
    className="alert__content"
  >
    <div
      className="usa-alert usa-alert--warning"
    >
      <div
        className="usa-alert__body"
      >
        <h3
          className="usa-alert__heading"
        >
          Update divert status?
        </h3>
        <p
          className="usa-alert__text"
        >
          Ambulances will be notified.
        </p>
      </div>
      <ul
        className="usa-button-group flex-column flex-align-stretch"
      >
        <li
          className="usa-button-group__item"
        >
          <button
            className="usa-button usa-button--outline bg-white width-full"
            onClick={[Function]}
            type="button"
          >
            Keep status
          </button>
        </li>
        <li
          className="usa-button-group__item"
        >
          <button
            className="usa-button usa-button--secondary width-full"
            onClick={[Function]}
            type="button"
          >
            Update status
          </button>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
